RELEASE_NOTES.rst

```diff
@@ -21,6 +21,43 @@
 
 .. towncrier release notes start
 
+Airflow 2.10.1 (2024-08-30)
+---------------------------
+
+Significant Changes
+^^^^^^^^^^^^^^^^^^^
+
+No significant changes.
+
+Bug Fixes
+"""""""""
+- Handle Example dags case when checking for missing files (#41874)
+- Fix logout link in "no roles" error page (#41845)
+- Set end_date and duration for triggers completed with end_from_trigger as True. (#41834)
+- DAGs are not marked as stale if the dags folder change (#41829)
+- Keep FAB compatibility for versions before 1.3.0 in 2.10 (#41809)
+- Don't Fail LocalTaskJob on heartbeat (#41810)
+- Remove deprecation warning for cgitb in Plugins Manager (#41793)
+- Fix log for notifier(instance) without __name__ (#41699)
+- Splitting syspath preparation into stages (#41694)
+- Adding url sanitization for extra links (#41680)
+- Fix InletEventsAccessors type stub (#41607)
+- Fix UI rendering when XCom is INT, FLOAT, BOOL or NULL (#41605)
+- Fix try selector refresh (#41503)
+- Incorrect try number subtraction producing invalid span id for OTEL airflow (#41535)
+- Add WebEncoder for trigger page rendering to avoid render failure (#41485)
+- Adding ``tojson`` filter to example_inlet_event_extra example dag (#41890)
+- Add backward compatibility check for executors that don't inherit BaseExecutor (#41927)
+
+Miscellaneous
+"""""""""""""
+- Bump webpack from 5.76.0 to 5.94.0 in /airflow/www (#41879)
+- Adding rel property to hyperlinks in logs (#41783)
+- Field Deletion Warning when editing Connections (#41504)
+- Remove Debian bullseye support (#41569)
```

Comment:
This is docs only for core. Which makes me wonder, there are no other docs-only changes for 2.10.1?

Reply:
Moved it to the Docs section. Also, there was a [PR](https://github.com/apache/airflow/commit/83aca021edb9054f43451171c7f415d5a6205536) which was not picked at all by `./dev/airflow-github changelog v2-3-stable v2-3-test` command, that too was docs only change.

Reply:
I have gone through other changes and I think we have two docs changes which I have listed under the Docs changes sections.